app/src/main/res/values/strings.xml

```diff
@@ -697,5 +697,6 @@
     <string name="clear_open_file_summary">Clears selected default file opening apps</string>
     <string name="legacy_listing_title">Use legacy listing for root</string>
     <string name="legacy_listing_summary">If enabled, uses stat for listing, else uses ls</string>
+    <string name="error_list_files">Error listing files at [%s]. It is possible that you don\'t have permission to access the directory.</string>
```

Comment:
Isn't this too big for a toast? specially with a path. 
My suggestion, maybe put a exclamation grey icon in empty view instead of no files. 
That solves the first half of sentence. The second half can then come as a toast.

Reply:
Great idea, but figuring out putting this grey exclamation in the file list view would take time that would past the 3.5.3 window...

Reply:
Suggest to log the stacktrace here and use `Permission denied` text. This text is too big for a toast of 2.5 secs.